tests/Doctrine/Migrations/Tests/Metadata/Storage/TableMetadataStorageTest.php

```diff
@@ -186,6 +187,67 @@ public function testCompleteWithFloatTime() : void
         ], $rows);
     }
 
+    public function testCompleteWillAlwaysCastTimeToInteger() : void
+    {
+        $schemaManager = $this->createMock(SqliteSchemaManager::class);
```

Comment:
Why choose to mock this concrete class rather than `AbstractSchemaManager`?

Reply:
Because I was debugging the test to create this mock and used the same used in the test, but just moved to the abstract one. I agree it is better. Thanks!